### PR TITLE
Added back button in AdminPanel page. Applied validation of admin for…

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
… invalid id and password. solved bug of ColumnConfig. Added few css for back button.

### CODE REVIEW

File: src/components/ColumnConfig.jsx [hunk 1]
```diff
@@ -28,59 +29,39 @@ class ColumnConfig extends Component{
...
-      let temporaryVisibleColumnConfig = this.props.visibleColumnConfig;
-      Object.keys(temporaryVisibleColumnConfig).forEach((key) => {
-        temporaryVisibleColumnConfig[key]= false;
-      });
-      this.setState({
-        visibleColumnConfig: temporaryVisibleColumnConfig,
-      });
+    }else {
+      if (temporarySelectValue === false) {
+        Object.keys(temporaryVisibleColumnConfig).forEach((key) => {
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]: Use for-in loop

File: src/components/DataGrid.jsx [hunk 2]
```diff
@@ -220,22 +229,21 @@ class DataGrid1 extends Component {
...
+      Edit</button>
     </div>
   );
-  componentDidMount() {
-    this.props.getAllStudentsAction();
-  }
+
   componentWillReceiveProps(nextProps){
-    if(nextProps.students!== this.props.students) {
+    if(nextProps.students !== this.props.students) {
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]: Use a proper deep comparison instead

File: src/components/ColumnConfig.jsx [hunk 3]
```diff
@@ -28,59 +29,39 @@ class ColumnConfig extends Component{
...
-    if(temporarySelectValue === false) {
-      let temporaryVisibleColumnConfig = this.props.visibleColumnConfig;
-      Object.keys(temporaryVisibleColumnConfig).forEach((key) => {
-        temporaryVisibleColumnConfig[key]= false;
-      });
-      this.setState({
-        visibleColumnConfig: temporaryVisibleColumnConfig,
-      });
+    }else {
+      if (temporarySelectValue === false) {
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/components/ColumnConfig.jsx]: Done.

File: src/components/ColumnConfig.jsx (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/ColumnConfig.jsx]: Done.

**Pull request merged**